go.mod

```diff
@@ -34,3 +34,5 @@ require (
 	gopkg.in/ini.v1 v1.52.0 // indirect
 	gopkg.in/yaml.v2 v2.2.4
 )
+
+replace github.com/minio/minio-go/v6 => ../minio-go
```

Comment:
temporary until https://github.com/minio/minio-go/pull/1262 gets merged

Reply:
It's merged

Reply:
and this line is removed